From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kefu Chai <kchai@redhat.com>
Date: Mon, 12 Apr 2021 17:28:56 +0800
Subject: [PATCH] common/Formatter: include used header
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

otherwise it fails to build with gcc-toolset-10, like:

../src/common/Formatter.cc: In member function ‘virtual void ceph::XMLFormatter::close_section()’:
../src/common/Formatter.cc:449:8: error: ‘transform’ is not a member of ‘std’
  449 |   std::transform(section.begin(), section.end(), section.begin(),
      |        ^~~~~~~~~

Signed-off-by: Kefu Chai <kchai@redhat.com>
---
 src/common/Formatter.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/common/Formatter.cc b/src/common/Formatter.cc
index b599e48b3..362deffb5 100644
--- a/src/common/Formatter.cc
+++ b/src/common/Formatter.cc
@@ -19,6 +19,7 @@
 #include "include/buffer.h"
 
 #include <fmt/format.h>
+#include <algorithm>
 #include <set>
 #include <limits>
 
